prow/cmd/phaino/README.md

```diff
@@ -33,11 +33,16 @@ volumes that the Prow Job may require.
 * `--print` the command that runs each job without running it
 * `--privileged` jobs are allowed to run instead of rejected
 * `--timeout=10m` controls how long to allow jobs to run before interrupting them
-* `--repo=k8s/test-infra=/go/src/k8s-test-infra` local path where prow job required repos are cloned at, can be passed in repeatedly
+* `--gopath=/go` overwrites the GOPATH to use in the container
```

Comment:
This seems odd to me, why is this helpful/necessary?

Reply:
I have removed the logic to overwrite the `GOPATH` env var, but it's still sometimes useful to allow users pass the `GOPATH` that will be used in the container. 

The reason we need it is when resolving the refs, the repos need to be cloned under the `GOPATH` (if they depend on it), but we cannot know what the `GOPATH` is without running the container, so we should allow users to tell where to clone the repo to.